### PR TITLE
Task 9 import from old fda

### PR DESCRIPTION
**Pull request by codymccoy**

### CODE REVIEW

File: ViewModel/Watershed/TerrainElement.cs [hunk 1]
```diff
@@ -49,15 +49,15 @@ public TerrainElement(string name, string fileName, bool isTemporaryNode = false
...
-                remove.Header = "Remove Terrain";
+                remove.Header = StringConstants.REMOVE_MENU;
                 remove.Action = RemoveElement;
 
                 Utilities.NamedAction renameElement = new Utilities.NamedAction(this);
-                renameElement.Header = "Rename";
+                renameElement.Header = StringConstants.RENAME_MENU;
                 renameElement.Action = Rename;
 
                 Utilities.NamedAction mapWindow = new Utilities.NamedAction();
```
Comment (RyanM-RMA) [inline ViewModel/Watershed/TerrainElement.cs:59]: Remove the unnecessary namespacing

**Review by RyanM-RMA — CHANGES_REQUESTED** [review]
Got some changes for you

**Review by RyanM-RMA — APPROVED** [review]

**Comment by RyanM-RMA** [comment]
Issue created to track the requested changes, Cody needs to update task 1 and 2, then fix task 9 changes.

File: Importer/AsciiImportExport/AsciiImport.cs [hunk 2]
```diff
@@ -61,27 +53,30 @@ public class AsciiImport : AsciiImportExport
...
+            ImportLevees,
+            ImportFrequency,
         }
 
         #endregion
         #region Properties
-        public DataTable StructuresForFDA2 { get; set; }
-        public List<IOccupancyType> OccupancyTypes { get; set; }
-        public List<WaterSurfaceElevationElement> WaterSurfaceElevs {get;set;}
+        private AsyncLogger _Logger;
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply Importer/AsciiImportExport/AsciiImport.cs:72]: i made it readonly

File: UserControls/TextBoxDescription.xaml [hunk 3]
```diff
@@ -35,8 +35,8 @@
             </Style>
         </Grid.Resources>
         <TextBox  Name="TxtDescription"  Text="{Binding Path=Text, UpdateSourceTrigger=PropertyChanged}" TextWrapping="Wrap" KeyDown="TxtDescription_KeyDown" VerticalAlignment="Stretch" HorizontalAlignment="Stretch"/>
-        <Button ToolTip="Expand" Style="{StaticResource TransparentStyle}" HorizontalAlignment="right" Margin="0,3,3,0" Name="Expand" VerticalAlignment="Top" Height="18" Width="18" Padding="0">
+        <!--<Button ToolTip="Expand" Style="{StaticResource TransparentStyle}" HorizontalAlignment="right" Margin="0,3,3,0" Name="Expand" VerticalAlignment="Top" Height="18" Width="18" Padding="0">
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply UserControls/TextBoxDescription.xaml:38]: Leaving for now, we might be adding this back in.

**Pull request merged**